README.md

```diff
@@ -1,24 +1,87 @@
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Check out our [Wiki](https://github.com/drink-this/drink-this-backend/wiki) for more info.
+<!-- features, example of the ML code, link to demo vid, link to wiki homepage -->
 
-Things you may want to cover:
+#### System Dependencies
+##### Some useful gems needed:
 
-* Ruby version
+###### Standard Rails Gems
+- `gem faraday`
+- `gem active_model_serializer`
+- `gem figaro`
 
-* System dependencies
+###### Using Python in Rails
+- `gem pycall`
+- `gem numpy`
+- `gem pandas`
 
-* Configuration
+We make use of three Python libraries commonly used in data science to construct our recommendation model: NumPy, pandas, and sci-kit learn (`sklearn` here). In order to use these inside of a Rails application, we utilize a gem called `PyCall`, which allows us to import and use these libraries, and by extension Python syntax and functions, within Ruby methods. The bulk of our model is constructed with Python, once we convert an array of Ruby objects (Ratings) from our database into a pandas DataFrame in our Dataframeable module:
 
-* Database creation
+<img width="784" alt="dataframeable" src="https://user-images.githubusercontent.com/26797256/128450438-682f1478-3820-4823-9ca1-3c089a0f9003.png">
 
-* Database initialization
+The sci-kit learn library gives us access to `metrics`, which includes score functions, performance metrics and pairwise metrics and distance computations, and by extension `pairwise`, from which we get the calculation of [Euclidean distance](https://en.wikipedia.org/wiki/Euclidean_distance). In Euclidean geometry, the Euclidean distance is the usual distance between two points (roughly, distance in multi-dimensional space), and this distance is measured as a line segment. In our model, we use the Euclidean distance to calculate, among users when compared to the current user, which users (the 15% of users in our database) is closest in preferences to the current user. This method enables our app to recommend an accurate cocktail to the current user based on the preferences of other users. 
 
-* How to run the test suite
+Some examples of this in action: 
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+<img width="844" alt="distance_calculations" src="https://user-images.githubusercontent.com/26797256/128450433-1f90c288-b48e-4174-9f6b-ff0c1feb12b0.png">
 
-* ...
+<img width="670" alt="weighted_ratings" src="https://user-images.githubusercontent.com/26797256/128450437-8794b1c0-1c6e-4869-ab72-0bf9217275d1.png">
+
+As the user base grows in our app (and thus our database of user ratings also grows), the recommendation model will become more accurate in providing a recommendation to a particular user that the they will enjoy.
```

Comment:
just a minor grammatical error `recommendation to a particular user that the they will enjoy.`
edit to `recommendation to a particular user that they will enjoy.`